### PR TITLE
Fix mobile versions downloads page

### PR DESCRIPTION
**Pull request by Bluewave2**
<pr>
Fixed it.
</pr>
<details>
<summary>Before</summary>

jruMvESfNO
</details>

<details>
<summary>After</summary>

CDaKUYIoWU
  
FPaWZy4rch
</details>

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -44,6 +44,16 @@ div .flexbox {
   justify-content: flex-start;
 }
 
+@media only screen and (max-width: 760px) and (orientation: portrait){
+  div .flexbox .box1 {
+    min-width: auto;
```
Comment (Chocobo1) [inline css/style.css:49]: why not apply them unconditionally? same below.

Comment (Bluewave2) [reply css/style.css:49]: I wanted to leave the original layout intact on larger screens while allowing smaller screens to have a usable layout. Didn't see a need to change it for larger screens.

**Comment by Chocobo1** [comment]
I made some minor adjustment and applied in 4c40bc94d444f01d52e0243c12a579ca2e5216b3. 

Thank you!

**Comment by Poopooracoocoo** [comment]
just out of curiosity, is there a way to fix it without using screen media queries? seems old fashioned.

**Pull request closed without merging**